api/envoy/config/metrics/v3/stats.proto

```diff
@@ -330,6 +330,14 @@ message DogStatsdSink {
   // Optional custom metric name prefix. See :ref:`StatsdSink's prefix field
   // <envoy_api_field_config.metrics.v3.StatsdSink.prefix>` for more details.
   string prefix = 3;
+
+  // Optional buffer size (in bytes) to use when sending UDP messages. By default
+  // Envoy will emit one metric per datagram. By specifying a buffer-size, Envoy
+  // will emit as many metrics that fit into the buffer. Metrics will be newline
+  // separated.
+  //
+  // Note that the buffer-size should not exceed your network's MTU.
+  int64 buffer_size = 4;
```

Comment:
Should this be called `max_bytes_per_datagram`? Buffer size seems to be an internal implementation details, so ideally we are describing effect here.

Reply:
Yeah, I think that's a good callout. I'm terrible at naming. :-P